### ISSUE TITLE
Pull requests here won’t count toward Hacktoberfest.

### ISSUE THREAD

**Issue by hacktoberfest-team** [RaidAndFade/Hacktoberfest2019#65]
Hi there,

Thank you for your interest in Hacktoberfest and in helping others make their first contributions to open source. 

While we agree that it's important to help others, this repository does not do this in a way that is in line with [the Hacktoberfest values](https://hacktoberfest.digitalocean.com/details). Please read the part that talks about high-quality contributions to understand why we are reaching out and taking action. _Due to this, we've added this repository to the list of excluded repositories which means pull requests here will not count toward Hacktoberfest 2020._

This is not a DigitalOcean decision, it's one that is inspired by the community of maintainers and contributors who all agree that quality is more important than quantity when it comes to engagement in the Open Source community. The values remind us to shift the focus from contributions to repositories that encourage folks to quickly create and gain a pull request to contributions that will help people level-up their skills and contribute to open-source projects that are in need of help. 

If you feel that this repository is an effective learning resource, we encourage you to keep running it since a core part of the aim for Hacktoberfest is to encourage new folks to get involved with open-source.

However, if you are interested in having contributions to this repo count toward Hacktoberfest we encourage you to take a look at this list of [Hacktoberfest issues](https://hacktoberfest.digitalocean.com/#projects) for inspiration and make the necessary changes to focus on enabling people to make meaningful contributions to open source projects.

If you have any questions about this or participating in Hacktoberfest, please contact our team via email [hacktoberfest@digitalocean.com](mailto:hacktoberfest@digitalocean.com) or join our [community Discord server](https://discord.gg/hacktoberfest).

Happy Hacking,
The Hacktoberfest Team

Hacktoberfest is presented by [DigitalOcean](http://hacktoberfest.digitalocean.com/), [Intel](https://intel.com/) and [DEV](https://github.com/forem/forem).

**Comment by RaidAndFade** [comment]
@hacktoberfest-team this repository specifically avoids the spam repository rule by challenging users to find unique solutions to interesting and uncommon problems. As you can see I don't allow spam prs to last very long if you look at prs that are meaningless or spam such as #33 or #23, in fact, any PR labeled "needs work" is automatically marked invalid after 48 hours to ensure that the authors of the request are active and want to be meaningful contributors. I assure you i am keeping the prs here up to the quality desired by Hacktoberfest this year. Perhaps even more than alot of other hacktoberfest repositories which just ask for menial tasks such as renaming files or adding new names to list. I do implore you to revise your decision here.

**Comment by RaidAndFade** [comment]
Tldr: this repository is not meant as a "quick pr" and those who take it as such are marked invalid. As they are spam. This repository tries to embody the hacktoberfest values by challenging users while also being simpler than an average PR to google/android or something like that

**Comment by RaidAndFade** [comment]
Note to all contributors: I will try to contact the team through their email, twitter, and facebook in order to get this resolved. 

Until then, it looks like your PRs will not be counted. Do what you will with that knowledge but I personally think the challenges, especially the ones coming in the next weeks, will be worth the time, whether or not they count it.

**Comment by RaidAndFade** [comment]
After many hours of discussion with their team, they have decided that this repository "doesn't align with their core values around quality contributions." And as such have kept this repository in their exclusion list. 

Because of this, I have rebranded the repository. Since they seem to have changed their core values in the past year, they used to wholeheartedly support repositories that provided good ways to learn the basics of OSS and Github, but clearly they are no longer like this. 

The tradition of my challenges will continue, with new challenges every sunday, but there will be disclaimers to show that your PR's may not count for hacktober, and personally i don't think that should matter. Use this repository as a stepping stone to the next one which will count. Just because you can't get one point here doesn't mean you can't get more elsewhere.
